kvrocks.conf

```diff
@@ -96,6 +96,16 @@ dir /tmp/kvrocks
 # We also can send logs to stdout/stderr is as simple as:
 #
 log-dir stdout
+# log-dir log
+
+# You can configure log-retention-days to control whether to enable logcleaner
+# and the maximum number of days that the INFO level logs will be kept.
+# if set to -1 , this mean disable logcleaner.
+# if set to between 0 to INT_MAX , this mean enable logcleaner ,
+# and INFO level log file whose last modified time is greater than log-retention-days will be unlinked.
+# if set to 0 , during the flush, the previous INFO level logs will be immediately unlinked.
+# By default the log-retention-days is -1.
+log-retention-days -1
```

Comment:
```suggestion

# You can configure log-retention-days to control whether to enable the log cleaner
# and the maximum retention days that the INFO level logs will be kept.
#
# if set to -1, that means to disable the log cleaner.
# if set to 0, all previous INFO level logs will be immediately removed.
# if set to between 0 to INT_MAX, that means it will retent latest N(log-retention-days) day logs. 

# By default the log-retention-days is -1.
log-retention-days -1
```